mmdet/apis/ote/apis/detection/ote_utils/label.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
I propose to move `label` from old SDK, or as Pavel proposed to add this to new SDK and remove it from old SDK

Reply:
Probably, it will be simpler just to take whole module from old sdk.